cmd/microcloud/main_init_preseed.go

```diff
@@ -463,59 +467,86 @@ func (p *Preseed) Parse(s *service.Handler, bootstrap bool, lookupTimeout time.D
 		}
 	}
 
+	// If an uplink interface was explicitly chosen, we will try to set up an OVN network.
+	explicitOVN := len(ifaceByPeer) > 0
```

Comment:
Can `ifaceByPeer` be > 1?

Reply:
It's the set of interfaces mapped by system name so it can be anything between 0 and `len(systemsBeingConfigured)`

Reply:
In particular, these are the uplink interfaces selected to use for the OVN uplink network. Configuring this is optional so the set can be empty.

Reply:
Ok thanks.